apps/framework-cli/src/cli/routines/docker_packager.rs

```diff
@@ -72,10 +72,24 @@ impl Routine for CreateDockerfile {
 
         ensure_docker_running()?;
 
-        let file_path = internal_dir.join("packager/Dockerfile");
+        let file_path = internal_dir.join("packager/versions/.gitkeep");
         let file_path_display = file_path.clone();
 
+        info!("Creating versions at: {:?}", file_path_display);
+        fs::create_dir_all(file_path.parent().unwrap()).map_err(|err| {
+            error!("Failed to create directory for app versions: {}", err);
+            RoutineFailure::new(
+                Message::new(
+                    "Failed".to_string(),
+                    "to create directory for app versions".to_string(),
+                ),
+                err,
+            )
+        })?;
+
         info!("Creating Dockerfile at: {:?}", file_path_display);
+        let file_path = internal_dir.join("packager/Dockerfile");
+        let file_path_display = file_path.clone();
```

Comment:
BTW, no need to `.clone`, just pass by reference instead of move.

`fs::write(&file_path`

Reply:
clone() is required. I tried pulling it out but needed due to:

```rust
 fs::create_dir_all(file_path.parent().unwrap()).map_err(|err| {
 ```